.gitignore

```diff
@@ -175,3 +175,73 @@ modules.xml
 /installer/deps
 
 # End of https://www.gitignore.io/api/phoenix
+
+# Created by https://www.gitignore.io/api/macos
```

Comment:
Same comment as the other PR (as well as a bunch of others). This file is getting out of hand.